### DISCUSSION TITLE
Displaying ID for visualization

### DISCUSSION THREAD

**Discussion by isaacebi** [talmolab/sleap#1686] [Help!]
Hello,

Is there any other way to display an ID or track name on the prediction? Currently, I am doing more than 10 tracks, and it's hard to track each individual ID via color as the colors tend to have the same color as the tracks exceed 10.

**Comment by Marielamend** [comment]
Hello @isaacebi!

Great question! We had another similar inquiry, #1667 to select tracks >10 in the GUI, hope this helps. 

In regards to having an enhancement in colors to differentiate the tracks is an excellent idea and I'll go ahead and mark this as an enhancement request!

Best, 
Mariela
